addons/sourcemod/scripting/surftimer/hooks.sp

```diff
@@ -899,7 +899,8 @@ public Action OnPlayerRunCmd(int client, int &buttons, int &impulse, float vel[3
 	}
 	else if (g_bInMaxSpeed[client])
 	{
```

Comment:
```
if(client){
return;
}

```
What about something like this? Possibly even as the first condition of this entire method?

Reply:
useless condition